lib/Mojolicious/Guides/Tutorial.pod

```diff
@@ -342,6 +342,32 @@ actions to templates.
 A list of all built-in ones can be found in
 L<Mojolicious::Plugin::DefaultHelpers> and L<Mojolicious::Plugin::TagHelpers>.
 
+=head2 Plugins
+
+Plugins are application extensions that help with code sharing and
+organization. You can load a plugin with the keyword
+L<Mojolicious::Lite/"plugin">, which can omit the C<Mojolicious::Plugin::> part
+of the name, and optionally provide configuration for the plugin.
```

Comment:
I think "which can omit the C<Mojolicious::Plugin::> part of the name" is out of place. Doesn't read very well to me. I think instead of mentioning which arguments `plugin()` can take, I would say *why* you want to write plugins. Something like "code sharing", "sharing logic between controllers", ...

Reply:
It is necessary to explain why `plugin Config` loads `Mojolicious::Plugin::Config`.

Reply:
If not documented here, I'd want to document it in either the Mojolicious::Lite/plugin or Mojolicious/plugin docs, neither of which currently contain this explanation of the arguments (just examples).

Reply:
Every plugin worth using documents how it prefers to be loaded. I don't think it adds much value for a tutorial context.

Reply:
That does not help with the other direction: seeing "plugin Config" in code and wanting to find out what that means.

Reply:
> It is necessary to explain why `plugin Config` loads `Mojolicious::Plugin::Config`.

I think it's up to Mojolicious::Plugin::Config to document how it is loaded. I do agree that it's useful to have the rules mention in a general section as well, but I don't think this is the right place.